v2/session/appinfo.mdx

```diff
@@ -4,14 +4,13 @@ title: About appInfo
 hide_title: true
 ---
 
+<!-- COPY DOCS -->
+<!-- ./thirdpartyemailpassword/appinfo.mdx -->
```

Comment:
This cannot use copy docs since some of the content doesn't apply to session recipe. Like "On the frontend, we need the domain for routing purposes, and on the backend, we need it so that we can generate correct email verification and password reset links."